### PR TITLE
update readme for 0.2.0 release

### PR DESCRIPTION
**Pull request by sofroniewn**
# Description
This PR updates our readme for our 0.2.0 release including screenshots with the new UI and some updates around our features.

It also contains a link to the imagesc forum and mentions the `napari` tag - this tag doesn't exist yet on their forum, but I'd like them to make it once we're actually signed up.

This PR still needs the release notes for `0.2.0` which I will generate after #546 gets merged.

It would be great if it could still be reviewed and approved now and then I will merge it just before making the `0.2.0` release

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -110,74 +88,50 @@ with napari.gui_qt():
...
     size = np.array([10, 20, 20])
     # add them to the viewer
-    points = viewer.add_points(points, size=size)
+    viewer.add_points(points, size=size)
 ```
 
 ![image](resources/screenshot-add-points.png)
 
-**napari** supports bidirectional communication between the viewer and the Python kernel, which is especially useful in Jupyter notebooks -- in the example above you can retrieve the locations of the points, including any additional ones you have drawn, by calling
+**napari** supports bidirectional communication between the viewer and the Python kernel, which is especially useful in Jupyter notebooks or from our built-in console. In this you can retrieve the locations of the points, including any additional ones you have drawn, by calling
```
Comment (kevinyamauchi) [inline README.md]: I think there should be a gif or annotated static image to show where to click to get the terminal.

File: README.md [hunk 2]
```diff
@@ -110,74 +88,50 @@ with napari.gui_qt():
...
 
-See [this issue](https://github.com/napari/napari/issues/301) for some of the features on the roadmap for our `0.2` release. Feel free to add comments or ideas!
+See [this issue](https://github.com/napari/napari/issues/420) for some of the features on the roadmap for our `0.3` release. Feel free to add comments or ideas!
 
 ## contributing
 
 Contributions are encouraged! Please read our [contributing guide](CONTRIBUTING.md) to get started. Given that we're in an early stage, you may want to reach out on [Github Issues](https://github.com/napari/napari/issues) before jumping in.
+
+## help
+We're a community partner on the [imagesc forum](https://forum.image.sc/tags/napari) and all help and support requests should be posted on the forum with the tag `napari`. We look forward to interacting with you there.
```
Comment (kevinyamauchi) [inline README.md]: 🎊

File: README.md [hunk 3]
```diff
@@ -110,74 +88,50 @@ with napari.gui_qt():
...
+You can render and quickly browse slices of multi-dimensional arrays in 2D or visualize them in 3D. Run `examples/nD_labels.py` to generate and interact with the following example:
 
-You can also paint pixel-wise labels, useful for creating masks for segmentation, and fill in closed regions using the paint bucket. Run `examples/add_labels.py` to generate and interact with the following example.
+![image](resources/screenshot-nD-slicing.gif)
 
-![image](resources/screenshot-add-labels.png)
+## tutorials
 
-For details checkout our [in-depth tutorials](https://napari.github.io/napari-tutorials/)
+For more details on how to use `napari` checkout our [in-depth tutorials](https://napari.github.io/napari-tutorials/)
```
Comment (kevinyamauchi) [inline README.md]: I think there should be a note here that these are a work in progress. Maybe something like:

```suggestion
For more details on how to use `napari` checkout our [in-depth tutorials](https://napari.github.io/napari-tutorials/). Please note that the tutorials are a work in progress. We are currently in the process of adding tutorials for each layer type.
```

**Review by kevinyamauchi — COMMENTED** [review]
Thanks, @sofroniewn ! I think this looks good.

I think we should clarify the usage of the `%gui qt5` magic in Jupyter notebooks. Perhaps this can go  under the "simple example".

File: README.md (hunk 1 above)
Comment (sofroniewn) [reply README.md]: i have turned the screenshot below into a gif

File: README.md (hunk 3 above)
Comment (sofroniewn) [reply README.md]: added

**Comment by sofroniewn** [comment]
@kevinyamauchi I have made you requested changes, including the gif of console opening, and clarifying the %gui qt5 magic command

**Comment by jni** [comment]
@sofroniewn sorry to contradict @kevinyamauchi but, as with the previous documentation PR, I'd like to avoid adding gifs in this repo, as they take up way too much memory (compare most screenshots added here at ~300KB vs both gifs weighing in at 1MB+). Could we have just screenshots here that (optionally) link to the tutorials repo?

File: README.md (hunk 2 above)
Comment (jni) [reply README.md]: Is this statement accurate or preempting some events that are yet to happen? =)

**Comment by sofroniewn** [comment]
@jni if i just delete the gifs now from this PR will they be permanently in our git record anyway? do i need to start a brand new pr and never add them, or will the squash merge take care of it?

@jni I'm leaning towards redoing the readme so that it only has one screenshot with the astronaut and then links straight to the tutorials, which I've been working on during my flight. I'll be making a PR there later tonight. Thoughts?

**Comment by sofroniewn** [comment]
i.e. in `features` only have text / maybe a list - no screenshots

File: README.md (hunk 2 above)
Comment (sofroniewn) [reply README.md]: to happen! but imagesc requests posting a message like this before they allow us to be community members

**Comment by kevinyamauchi** [comment]
> @sofroniewn sorry to contradict @kevinyamauchi but, as with the previous documentation PR, I'd like to avoid adding gifs in this repo, as they take up way too much memory (compare most screenshots added here at ~300KB vs both gifs weighing in at 1MB+). Could we have just screenshots here that (optionally) link to the tutorials repo?

Good point. This totally makes sense to me!

**Comment by jni** [comment]
@sofroniewn yes, deleting the files should keep them off the history when the commits get squashed, as the squash commit will only contain the differences between master and the final state of this PR.

**Comment by jni** [comment]
And I love the idea of the single screenshot! I even thought about suggesting it but thought it might be rebuked as "too radical". =P

**Comment by sofroniewn** [comment]
Ok @jni this has been updated dropping the gifs and having just a single screenshot. I much prefer the readme now - I think we could even drop the `$ pip install git+https://github.com/napari/napari` option - we have that in our in-depth installation tutorial and I'd rather make that section a little shorter. Thoughts?

**Comment by jni** [comment]
> I think we could even drop the $ pip install git+https://github.com/napari/napari option

Go for it! imho those that could be encouraged to use that option will know enough to think of it themselves. I think that was useful when our master was constantly much better than the released version, but it is much less needed once 0.2 is out.

**Review by jni — APPROVED** [review]

**Comment by sofroniewn** [comment]
I've updated this with new buttons for reseting the view, transposing, and rolling the dimensions as discussed with @jni. We might want to improve the icon design at some point, but it's probably fine for now.

I will merge once the tests pass and then make the `0.2.0` release

**Comment by imagejan** [comment]
Great to see you on the image.sc forum! I suggest adding a badge linking to the forum, I submitted a PR to this topic branch: https://github.com/sofroniewn/napari/pull/1.

Preview:

[image.sc forum](https://forum.image.sc/tags/napari)

**Comment by sofroniewn** [comment]
Thanks @imagejan - that's a great addition, and you got it in just in time ;-) will be merging this after the test pass :-)

**Pull request merged**